src/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```

Comment:
You have the whole project in javaScript, but this file in TypeScrypt (at least `.tsx` file format). Is it new religion?)